### PR TITLE
Create ParalyzeMod.java

### PR DESCRIPTION
**Pull request by mcccs**

### CODE REVIEW

**Comment by Eyremba** [comment]
You are using 1337 movement-packets, but this does not always work. I would recommend a ".paralyzepackets" command to change it. Some client also use this with values up to ~90000

**Comment by mcccs** [comment]
I don't know how to do this.

File: Wurst Client/src/tk/wurst_client/mods/ParalyzeMod.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ */
+package tk.wurst_client.mods;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.entity.EntityPlayerSP;
+import net.minecraft.client.network.NetHandlerPlayClient;
+import net.minecraft.network.play.client.C03PacketPlayer;
+//import saint.eventstuff.Event;
+//import saint.eventstuff.events.PreMotion;
+//import saint.modstuff.Module;
```
Comment (Alexander01998) [inline Wurst Client/src/tk/wurst_client/mods/ParalyzeMod.java]: Please...

**Comment by Alexander01998** [comment]
Although it would probably work, I can't accept it like that. Call it ethical reasons, I don't know.

**Pull request closed without merging**